File: LeetCode-Problems/Easy/Two-Sum-II.java
//Java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int[] rArr = new int[2];
        for (int i = 0; i < numbers.length; i++){
            for (int j = i+1; j < numbers.length; j++){
                if (numbers[i] + numbers[j] == target){
                    rArr[0] = i+1;
                    rArr[1] = j+1;
                    return rArr;
                }
            }
        }
        return rArr;
    }
}

File: LeetCode-Problems/Easy/Two-Sum-II.py
# Python
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        rArr = list()
        for x in range(0, len(numbers)):
            for y in range(x+1, len(numbers)):
                if (numbers[x] + numbers[y] == target):
                    rArr.append(x+1)
                    rArr.append(y+1)
                    return rArr
        return rArr
        
